app/components/efl_qualification_card_component.rb

```diff
@@ -0,0 +1,32 @@
+class EflQualificationCardComponent < ViewComponent::Base
+  attr_reader :application_form
+
+  def initialize(application_form)
+    @application_form = application_form
+  end
+
+  def show_component
```

Comment:
This is built-in! https://github.com/github/view_component#conditional-rendering

Reply:
Nice, thanks for the link! Will rework this.